Data/cfd/readme.md

```diff
@@ -0,0 +1,80 @@
+![alt tag](https://raw.githubusercontent.com/QuantConnect/Lean/master/Documentation/logo.white.small.png) 
+## LEAN Data Formats / Contract for difference (cfd)
+
+QuantConnect hosts cfd data provided by Oanda. The data contains only *Quote* data.
+All data are stored in zip files, each containing a single CSV file. You can explore cfd data on our website at https://www.quantconnect.com/data/tree/cfd
+
+Cfd data supports the following Resolutions:
+
+* Tick
+* Second
+* Minute
+* Hour
+* Daily
+
+The markets we currently support are:
+
+* Oanda - Opens `Sunday 17:00:00` Closes `Friday 17:00:00` EST / UTC-05
+
+All times are in UTC unless noted otherwise.
+
+## Tick Format
```

Comment:
Please put `Second, Minute` first, Hour,Daily next; then tick last; following styling of the existing Equity page:
https://github.com/QuantConnect/Lean/blob/master/Data/equity/readme.md#minute-second-data-file-format